cibuildwheel/__main__.py

```diff
@@ -67,6 +68,19 @@ def main() -> None:
                             run in Windows, and it will build and test for all versions of
                             Python. Default: auto.
                         ''')
+
+    parser.add_argument(
+        '--archs',
+        default=None,
+        help='''
+            Comma-separated list of CPU architectures to build for.
+            If unspecified, builds the architectures natively supported
+            on this machine. Set this option to build an architecture
+            via emulation, for example, using binfmt_misc and qemu.
+            Default: auto
```

Comment:
I discussed both names then picked `auto` because it is used by `--plat`, just above. It might still be better to make this `native`, and not match `--plat`, though.

Reply:
Up to @joerick, as far as I'm concerned. I just found it striking, reading this PR's code for the first time, that every time there's `auto`, there's some kind of explanation that "auto means the native architectures, here".

Reply:
I find `auto` a bit better than `native` because, imho, `native` implies a single architecture whereas `auto` suggests something more.

Reply:
My issue with `auto` is that it sounds like "you detect which architectures I can build".

If `native` only implies one architecture (so... no `i686` on `x86_64` Linux, I assume?), then "If unspecified, builds the architectures natively supported on this machine." is an incorrect description, though.

But yes, I was just bikeshedding here; there's nothing else to complain about, so ... ;-)